x-pack/plugins/ml/public/explorer/explorer.js

```diff
@@ -708,26 +764,36 @@ export const Explorer = injectI18n(injectObservablesAsProps(
         stateUpdate.influencers = await loadTopInfluencers(jobIds, timerange.earliestMs, timerange.latestMs, noInfluencersConfigured);
       }
 
+      if (stateUpdate.influencers !== undefined && !noInfluencersConfigured) {
+        for (const influencerName in stateUpdate.influencers) {
+          if (stateUpdate.influencers[influencerName][0] && stateUpdate.influencers[influencerName][0].influencerFieldValue) {
```

Comment:
Great question! So the return from `loadTopInfluencers`  `contains an influencers property, with a key for each of the influencer field names, whose value is an array of objects containing influencerFieldValue, maxAnomalyScore and sumAnomalyScore keys`
https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/public/services/results_service.js#L264

Since this is just to create the placeholder for the filter bar we only need one example of a field name/value pair to show the syntax - that's why I'm just hardcoding for the first element and breaking out once we get something that has a field value.